lib/rdkafka/bindings.rb

```diff
@@ -161,6 +165,33 @@ class TopicPartitionList < FFI::Struct
       end
     end
 
+    # Callbacks are currently global and contextless.
+    # The same callback object is assigned for statistics, errors, and potentially OAuth regardless of the instance to which it is associated.
```

Comment:
This is not true.

The same statistics-dedicated callback will be assigned to all instances, but not the same callback will be assigned to statistics, errors, and OAuth.

This is not the same. Would you mind correcting this?

Reply:
I was going off of your comment at https://github.com/karafka/rdkafka-ruby/pull/410#issuecomment-1970876846.  I guess I misinterpreted your comment.  Will fix.